apps/civicsignalblog/src/payload/collections/CivicSignalBlog/Pages.js

```diff
@@ -0,0 +1,31 @@
+import canRead from "../../access/applications/researchBlog";
+import CustomPageHeader from "../../blocks/CustomPageHeader";
+import Error from "../../blocks/Error";
+import FeaturedStories from "../../blocks/FeaturedStories";
+import LongForm from "../../blocks/LongForm";
+import PageHeader from "../../blocks/PageHeader";
+import Posts from "../../blocks/Posts";
+import pages from "../../fields/pages";
```

Comment:
I don't think `pages` is a field... may be just a function in `utils`?

Reply:
Done I have moved it to utils